share/completions/git.fish

```diff
@@ -191,19 +191,29 @@ complete -r -c git -n '__fish_git_using_command filter-branch' -s d -d 'Use this
 complete -c git -n '__fish_git_using_command filter-branch' -s f -l force -d 'Force filter branch to start even w/ refs in refs/original or existing temp directory'
 
 ### remote
+set -l remotecommands add rm show prune update rename set-head set-url set-branches
 complete -f -c git -n '__fish_git_needs_command' -a remote -d 'Manage set of tracked repositories'
 complete -f -c git -n '__fish_git_using_command remote' -a '(__fish_git_remotes)'
-complete -f -c git -n '__fish_git_using_command remote' -s v -l verbose -d 'Be verbose'
-complete -f -c git -n '__fish_git_using_command remote' -a add -d 'Adds a new remote'
-complete -f -c git -n '__fish_git_using_command remote' -a rm -d 'Removes a remote'
-complete -f -c git -n '__fish_git_using_command remote' -a show -d 'Shows a remote'
-complete -f -c git -n '__fish_git_using_command remote' -a prune -d 'Deletes all stale tracking branches'
-complete -f -c git -n '__fish_git_using_command remote' -a update -d 'Fetches updates'
-complete -f -c git -n '__fish_git_using_command remote' -a rename -d 'Renames a remote'
-complete -f -c git -n '__fish_git_using_command remote' -a set-head -d 'Sets the default branch for a remote'
-complete -f -c git -n '__fish_git_using_command remote' -a set-url -d 'Changes URLs for a remote'
-complete -f -c git -n '__fish_git_using_command remote' -a set-branches -d 'Changes the list of branches tracked by a remote'
-# TODO options
```

Comment:
@faho I removed this TODO since this commit adds a handful of options. I am not sure if that is OK with so feel free to let me know and I will revert this change